src/shared/registered-app.component.js

```diff
@@ -0,0 +1,101 @@
+import React, { useContext } from "react";
+import { useCss } from "kremling";
+import Code from "./code.component";
+import { LocalStorageContext } from "./use-local-storage-data.hook";
+import { ApplicationBundleLink } from "./links.component";
+
+export default function RegisteredApp({ app, edit, interactive }) {
+  const scope = useCss(css);
+
+  const { removeApplication } = useContext(LocalStorageContext);
+
+  return (
+    <div {...scope}>
+      <div className="registered-app">
+        <Code code={indentedCode(app)} />
+        {interactive && (
+          <>
+            <div role="button" tabIndex={0} onClick={edit} className="edit">
```

Comment:
Could these become proper `button`s instead? Divs with onClick handlers are inaccessible because a keyboard user cannot interact with it.